ui/components/component-library/avatar-icon/avatar-icon.types.ts

```diff
@@ -0,0 +1,35 @@
+import { IconColor, TextColor } from '../../../helpers/constants/design-system';
+import { IconName, IconProps } from '../icon';
+import { AvatarBaseProps } from '../avatar-base/avatar-base.types';
+
+export enum AvatarIconSize {
+  Xs = 'xs',
+  Sm = 'sm',
+  Md = 'md',
+  Lg = 'lg',
+  Xl = 'xl',
+}
+
+export interface AvatarIconProps extends Omit<AvatarBaseProps, 'color'> {
```

Comment:
Can we also omit `children` so we don't have to pass `undefined` as a prop. The `<Icon />` component should suffice as the children prop

Reply:
OKay, I will do that.